lldb/source/Utility/ArchSpec.cpp

```diff
@@ -353,6 +353,8 @@ static const ArchDefinitionEntry g_macho_arch_entries[] = {
     {ArchSpec::eCore_x86_64_x86_64,   llvm::MachO::CPU_TYPE_X86_64,     llvm::MachO::CPU_SUBTYPE_X86_ARCH1,     UINT32_MAX, SUBTYPE_MASK},
     {ArchSpec::eCore_x86_64_x86_64h,  llvm::MachO::CPU_TYPE_X86_64,     llvm::MachO::CPU_SUBTYPE_X86_64_H,      UINT32_MAX, SUBTYPE_MASK},
     {ArchSpec::eCore_x86_64_x86_64,   llvm::MachO::CPU_TYPE_X86_64,     CPU_ANY, UINT32_MAX, UINT32_MAX},
+    {ArchSpec::eCore_riscv32,         llvm::MachO::CPU_TYPE_RISCV,      llvm::MachO::CPU_SUBTYPE_RISCV_ALL,     UINT32_MAX, SUBTYPE_MASK},
```

Comment:
I don't know this stuff works. Do we need riscv64 too or is it shared?

Reply:
There isn't a CPU_TYPE for riscv64.

Reply:
Should we name this eCore_riscv then? Given that it applies to both.

Reply:
That's not quite accurate. It doesn't apply to both, it _is_ riscv32.

Reply:
Oh so this is just how it's been named? If you're matching what the MachO spec (whatever form it is) then that's fine, too late to change it now.

And I take from that that at some point a riscv64 value could be allocated for MachO but there is not one at the moment.

Reply:
Yeah, I can't comment on the future but that's correct.

Reply:
Cool. Conclusion here is that in this context, CPU_TYPE_RISCV means riscv32. I might have added a 32 on the end there, but it is what it is :)